packages/@aws-cdk/aws-cloudfront/lib/distribution.ts

```diff
@@ -658,30 +659,22 @@ export interface AddBehaviorOptions {
   readonly cachedMethods?: CachedMethods;
 
   /**
-   * Whether you want CloudFront to automatically compress certain files for this cache behavior.
-   * See https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/ServingCompressedFiles.html#compressed-content-cloudfront-file-types
-   * for file types CloudFront will compress.
+   * The cache policy for this behavior. The cache policy determines what values are included in the cache key,
+   * and the time-to-live (TTL) values for the cache.
+   * See https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/controlling-the-cache-key.html.
```

Comment:
```suggestion
   * @see https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/controlling-the-cache-key.html.
```

unless there was more that got truncated

Reply:
This is interesting! Our "generic" docs drop `@see` lines altogether (example: https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-cloudfront.DistributionProps.html#webaclid-span-class-api-icon-api-icon-experimental-title-this-api-element-is-experimental-it-may-change-without-notice-span), whereas all of the language-specific docs include them (albeit in a pretty ugly way half the time). I'll update.